includes/polling/ports.inc.php

```diff
@@ -578,10 +587,14 @@
         if ($port_association_mode != "ifIndex") {
             $port['update']['ifIndex'] = $ifIndex;
         }
-
-        $port['update']['poll_time'] = $polled;
-        $port['update']['poll_prev'] = $port['poll_time'];
-        $port['update']['poll_period'] = $polled_period;
+        
+        // set $port['update'] times if different from $device_global_ports
+        // if $port['update'] times not set and $port['update'] is void, value would be replaced by $device_global_ports
```

Comment:
I still don't follow why this code block is necessary.  Isn't that the reason we build $port_ids.  (also $port_ids variable name could be more descriptive)